src/_data/whats-new.yml

```diff
@@ -4,8 +4,208 @@ description: |
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Mon Mar 30 08:59:14 2020
+updated: Wed Apr 22 10:36:00 2020
 entries:
+- description: Added 'Set a fixed price for bundled products' to the [REST usage notes](https://devdocs.magento.com/guides/v2.3/rest/notes.html)
+    topic.
+  versions: 2.3.x
+  type: Major Update
+  date: April 20, 2020
+  link: https://github.com/magento/devdocs/pull/7035
+- description: Added correct db schema information in the [Configure declarative schema](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/declarative-schema/db-schema.html)
+    topic.
+  versions: 2.3.x
+  type: Technical
+  date: April 16, 2020
+  link: https://github.com/magento/devdocs/pull/7045
+- description: Added a new reference topic for the [system.xml](https://devdocs.magento.com/guides/v2.3/config-guide/prod/config-reference-systemxml.html)
+    configuration file.
+  versions: 2.3.x
+  type: New Topic
+  date: April 16, 2020
+  link: https://github.com/magento/devdocs/pull/7024
+- description: 'New Page Builder topic on how to configure three different options
+    for styling elements in your content types: [How to configure styling options](https://devdocs.magento.com/page-builder/docs/how-to/how-to-configure-styling-options.html).'
+  versions: 2.3.x
+  type: Technical
```

Comment:
This data does not correspond to data in linked PR https://github.com/magento/devdocs/pull/7044

This file should not be edited manually. It must be regenerated after all the fixes are done to corresponding PRs by assigned writers.

I did not check other entries.

Reply:
The description matches, but the label should probably be New Topic.

Reply:
@bdenham should #7044 have a New Topic label? Judging by the description, it looks like it's just an update to the link text and destination

Reply:
By `this data`, I meant type. The original PR contains two labels for type _Major Update_ and _Technical_. But here I see only _Technical_. It means it was edited manually, not regenerated. Entries here must reflect real state of PRs. If something is wrong there, it must be edited there, because if we edit it here, we are merely hiding the issue. PRs are still available even after they were merged. They can be used in multiple searches by text, by labels, by version, etc. This way allows us to pay more attention to whether PRs are labeled correctly, because we do not have automation to do this right there on GitHub.

Reply:
Ah, yeah, this was a weird one. Long story short, the new topic label would be appropriate. I’ll get up and change it as soon as I can.

Reply:
Ah, yeah, this was a weird one. Long story short, the new topic label would be appropriate. I’ll get up and change it as soon as I can.

Reply:
I just changed the original PR labeling to New Topic.

![image](https://user-images.githubusercontent.com/1828494/80140445-8fe06a00-856d-11ea-9a00-5a1bfcfdf49e.png)

Change `type` to New Topic, and you should be fine.